client/signup/steps/woocommerce-install-confirm-wordpress-subdomain/index.tsx

```diff
@@ -0,0 +1,141 @@
+import {
+	fetchAutomatedTransferStatusOnce,
+	requestEligibility,
+} from 'calypso/state/automated-transfer/actions';
+import {
+	isFetchingAutomatedTransferStatus,
+	getEligibility,
+	EligibilityData,
+} from 'calypso/state/automated-transfer/selectors';
+import { getSiteDomain } from 'calypso/state/sites/selectors';
+import { getSelectedSiteId } from 'calypso/state/ui/selectors';
+import type { GoToStep } from '../../types';
+import type { AppState } from 'calypso/types';
+
+import './style.scss';
+interface Props {
+	goToStep: GoToStep;
+	isReskinned: boolean;
+}
+
+export default function ConfirmWordPressSubdoamin( {
+	goToStep,
+	isReskinned,
+}: Props ): ReactElement | null {
+	const siteId = useSelector( getSelectedSiteId ) as number;
+
+	const fetchingTransferStatus = !! useSelector( ( state: AppState ) =>
+		isFetchingAutomatedTransferStatus( state, siteId )
+	);
+
+	const dispatch = useDispatch();
+	useEffect( () => {
+		if ( ! siteId ) {
+			return;
+		}
+		dispatch( fetchAutomatedTransferStatusOnce( siteId ) );
+		dispatch( requestEligibility( siteId ) );
+	}, [ siteId, dispatch ] );
+
+	const wpcomDomain = useSelector( ( state: AppState ) => getSiteDomain( state, siteId ) );
+	const stagindDomaon = wpcomDomain?.replace( /\b.wordpress.com/, '.wpcomstaging.com' );
+
+	const { eligibilityWarnings }: EligibilityData = useSelector( ( state ) =>
+		getEligibility( state, siteId )
+	);
+
+	const wordPressSubdomainWarning =
+		eligibilityWarnings && eligibilityWarnings.find( ( { id } ) => id === 'wordpress_subdomain' );
+
+	// Check whether it should skip the Wordpress Subdomain warning step
+	useEffect( () => {
+		if ( ! siteId ) {
+			return;
+		}
+
+		if ( fetchingTransferStatus ) {
+			return;
+		}
+
+		if ( wordPressSubdomainWarning ) {
+			return;
+		}
+
+		goToStep( 'confirm' ); // skip!
+	}, [ wordPressSubdomainWarning, fetchingTransferStatus, goToStep, siteId ] );
+
+	const headerTitle = __( 'Your new store' );
+	const headerDescription = (
+		<>
+			{ __( 'This will be your new store domain.' ) }
+			<br />
+			{ __( 'You can change it later and get a custom one.' ) }
+		</>
+	);
+
+	const isProcessing = ! siteId || fetchingTransferStatus || ! wordPressSubdomainWarning;
+
+	function getStepContent() {
+		return (
+			<>
+				<div className="woocommerce-install-confirm-wordpress-subdomain__image-container">
+					<img src={ yourNewStoreImage } alt="" />
+				</div>
+				<div className="woocommerce-install-confirm-wordpress-subdomain__instructions-container">
+					<div className="woocommerce-install-confirm-wordpress-subdomain__instructions-wpcom-domain">
+						{ wpcomDomain }
+					</div>
+					<div className="woocommerce-install-confirm-wordpress-subdomain__instructions-staging-domain">
+						{ stagindDomaon }
+					</div>
+
+					<p>
+						{ __(
+							'By installing this product your subdomain will change. You can change it later to a custom domain and we will pick up the tap for a year.'
```

Comment:
Looks like there may have been a typo in the designs here, should it be "we will pick up the tab for a year"?

Reply:
probably? I copied and p[asted from the design file. Checking...

Reply:
addressed here. cc @ianstewart